golang/go.mod

```diff
@@ -1,10 +1,29 @@
 module github.com/kurtosis-tech/minimal-grpc-server/golang
 
-go 1.15
+go 1.18
 
 require (
+	github.com/improbable-eng/grpc-web v0.15.0
 	github.com/kurtosis-tech/stacktrace v0.0.0-20211028211901-1c67a77b5409
-	github.com/sirupsen/logrus v1.4.1
-	github.com/stretchr/testify v1.7.0
-	google.golang.org/grpc v1.35.0
+	github.com/sirupsen/logrus v1.7.0
+	github.com/soheilhy/cmux v0.1.5
```

Comment:
do we know what the trade offs here are?

Reply:
yepp;

without cmux - would require to expose 2 or more ports... and hooking that needs some plumbing. we can look into connect.